docs/quickstart/apollo_2_5_hardware_system_installation_guide_v1.md

```diff
@@ -0,0 +1,1004 @@
+        * [Installing the GPS Receiver and Antenna](#installing-the-gps-receiver-and-antenna)
+        * [Installing the Light Detection and Ranging System (LiDAR)](#installing-the-light-detection-and-ranging-system-lidar)
+        * [Installing the Cameras](#installing-the-cameras)
+        * [Installing the Radar](#installing-the-radar)
+        * [Installing the IPC](#installing-the-ipc)
+        * [Configuring the GPS and IMU](#configuring-the-gps-and-imu)
+* [Setting up the Network](#setting-up-the-network)
+    * [Recommendations](#recommendations)
+* [Additional Tasks Required](#additional-tasks-required)
+* [Next Steps](#next-steps)
+
+
+# About This Guide
+
+The *Apollo 2.5 Hardware and System Installation Guide* provides the instructions to install all of the hardware components and system software for the **Apollo Project**. The system installation information included pertains to the procedures to download and install the Apollo Linux Kernel.
+
+## Document Conventions
+
+The following table lists the conventions that are used in this document:
+
+| **Icon**                            | **Description**                          |
+| ----------------------------------- | ---------------------------------------- |
+| **Bold**                            | Emphasis                                 |
+| `Mono-space font`                   | Code, typed data                         |
+| _Italic_                            | Titles of documents, sections, and headings Terms used |
+| ![info](images/info_icon.png)       | **Info**  Contains information that might be useful.  Ignoring the Info icon has no negative consequences. |
+| ![tip](images/tip_icon.png)         | **Tip**. Includes helpful hints or a shortcut that might assist you in completing a task. |
+| ![online](images/online_icon.png)   | **Online**. Provides a link to a particular web site where you can get more information. |
+| ![warning](images/warning_icon.png) | **Warning**. Contains information that must **not** be ignored or you risk failure when you perform a certain task or step. |
+
+# Introduction
+
+The **Apollo Project** is an initiative that provides an open, complete, and reliable software platform for Apollo partners in the automotive and autonomous driving industries. The aim of this project is to enable these entities to develop their own self-driving systems based on Apollo software stack.
+
+## Documentation
+
+The following set of documentation describes Apollo 2.5:
+
+- ***<u>[Apollo Hardware and System Installation Guide]</u>***  ─ Provides the instructions to install the hardware components and the system software for the vehicle:
+
+    - **Vehicle**:
+
+      - Industrial PC (IPC)
+      - Global Positioning System (GPS)
+      - Inertial Measurement Unit (IMU)
+      - Controller Area Network (CAN) card
+      - GPS Antenna
+      - GPS Receiver
+      - Light Detection and Ranging System (LiDAR)
+      - Camera
```

Comment:
Camera --> Cameras

Reply:
Should we put Pandora related information into 2.5 Guide?